robot.py

```diff
@@ -3,11 +3,16 @@
 
 import drivestation
 import driveteam
+
+import swerve.swervesubsystem
+import utils.utils
+
 from shuffleboard import addDeployArtifacts
 
 
 class MyRobot(wpilib.TimedRobot):
   pilots = driveteam.DriveTeam()
+  drivebase = swerve.swervesubsystem.SwerveSubsystem()
```

Comment:
Initialise pilots and drivebase in an init function as instance attributes "e.g. self.drivebase" instead of class attributes.